libs/datamodel/core/src/configuration/mod.rs

```diff
@@ -14,15 +14,7 @@ pub struct Configuration {
 #[serde(rename_all = "camelCase")]
 #[derive(Clone, Debug, Serialize)]
 pub struct StringFromEnvVar {
```

Comment:
Taking a closer look at https://github.com/prisma/prisma-engines/pull/800#discussion_r447601367, it's still unclear to me how these values relate to each other. When is `from_env_var` populated? When is `value` populated? There seems to be some intent behind how to construct and use these and it's unclear to me what that is.

Reply:
Does the line below not clarify that?

Reply:
On re-reading it I guess it's fine, though I was hoping it could be clearer. We now have a struct member communicating intent over the whole struct which is something I would expect to live at the top level of the struct declaration site.

But I feel like we've talked quite a bit about this patch already; feel free to merge this PR as-is (: